.gitignore

```diff
@@ -9,3 +9,5 @@ test/*.js
 ~test/jest.config.js
 !test/tests/**/.eslintrc*
 !test/tests/**/*.log
+#VS Code workspace file
+*.code-workspace
```

Comment:
Err this and the PR title look unrelated. Could you please file a separate issue if you'd like this change?

Reply:
My apologies for the delay... I went out of town and had no service so was pretty disconnected, glad to see it got taken care of. Well, if you would like to have code-workspaces ignored I'm not opposed to resbubmitting.